hazelcast/src/main/java/com/hazelcast/jet/impl/util/ExceptionUtil.java

```diff
@@ -227,4 +231,12 @@ public static boolean isOrHasCause(Throwable t, Class<?> classToFind) {
         }
         return t != null && classToFind.isAssignableFrom(t.getClass());
     }
+
+    public static boolean isTechnicalCancellationException(Throwable t) {
+        Throwable peeledFailure = peel(t);
+        return peeledFailure instanceof CancellationException
```

Comment:
I'm still not sure about ```CancellationException```. We can throw it during restarting of one member in the cluster.

Reply:
You will still have traces in the logs, but without exception (that didn't provide any additional information)

Reply:
I can't put together a good argument why it shouldn't be logged, so removed for now cd7aac3314e